passthrough-server/src/main/java/org/terracotta/passthrough/SimpleClientTestEnvironment.java

```diff
@@ -47,4 +48,9 @@ public int getTotalClientCount() {
   public int getThisClientIndex() {
     return this.thisClientIndex;
   }
+
+  @Override
+  public IClusterInfo getClusterInfo() {
```

Comment:
This should probably just be expanded so you don't need a new implementation of this interface, in Galvan.  In hindsight, even creating the IClientTestEnvironment interface was probably overkill (as it is just a read-only container of data - no behavior).